### ISSUE TITLE
Can't change filter type in Flux builder

### ISSUE THREAD

**Issue by chnn** [influxdata/chronograf#4484]
![can t change filter type to](https://user-images.githubusercontent.com/638955/45637025-4610d300-ba5e-11e8-9b55-f2f6a5098c2a.gif)

**Comment by AlirieGray** [comment]
blocked on: https://github.com/influxdata/flux/issues/81

**Comment by AlirieGray** [comment]
Separate issue: filter tag item closing when you click the "=/!=" radio button
